### ISSUE TITLE
Declaration of OnApp_VirtualMachine::getList()

### ISSUE THREAD

**Issue by luceos** [virtuozzo/OnApp-PHP-Wrapper-External#3]
`Error (Declaration of OnApp_VirtualMachine::getList() should be compatible with that of OnApp::getList()) in file /cms/lib/hyn/onapp/lib/VirtualMachine.php [932] with context Array`

Only an issue on PHP 5.3+

**Comment by yuriy-yakubskiy** [comment]
Hi Luceos,

Thank you for reporting could you please tell the way to reproduce the issue, because it works like this without errors.
Tested on PHP Version 5.3.2-1ubuntu4.14

```
<?php
error_reporting( E_ALL );
ini_set( 'display_errors', 1 );
ini_set('html_errors', 1);

require_once 'wrapper/OnAppInit.php';

$onapp = new OnApp_Factory('host', 'user', 'password');

$vm = $onapp->factory( 'VirtualMachine', true );

$vm->getList();

print('<pre>');
print_r($vm);

die();
```

Thanks,
Yuriy Yakubskiy
OnApp PHP Dev Team

**Comment by luceos** [comment]
Oh yes, it's probably only a notice but thrown because in VirtualMachine.php:

`function getList( $user_id = NULL ) {`

Is not an equal declaration as in Onapp.php:

`public function getList( $params = null, $url_args = null ) {`

Which is not accepted in PHP 5.3+, so the definition in VirtualMachine must match that of OnApp.

(and ps, thank you so much for such a comprehensible library, even with only 6 watchers, this library is absolutely a necessity for onapp to have) :)

**Comment by OnApp** [comment]
Fixed in branch 3.0.0.
